src/yuzu/main.cpp

```diff
@@ -243,6 +243,33 @@ GMainWindow::GMainWindow()
     const auto yuzu_build_version = override_build.empty() ? yuzu_build : override_build;
 
     LOG_INFO(Frontend, "yuzu Version: {}", yuzu_build_version);
+
+#ifdef _MSC_VER
+    // It is possible that the name of the dll will change.
+    // vcruntime140.dll is for 2015 and onwards
+    const char* runtime_dll_name = "vcruntime140.dll";
+    DWORD dwHandle, sz = GetFileVersionInfoSizeA(runtime_dll_name, &dwHandle);
+    bool runtime_version_inspection_worked = FALSE;
+    if (sz > 0) {
+        std::vector<unsigned char> buf(sz);
+        if (GetFileVersionInfoA(runtime_dll_name, dwHandle, sz, &buf[0])) {
+            VS_FIXEDFILEINFO* pvi;
+            sz = sizeof(VS_FIXEDFILEINFO);
+            if (VerQueryValueA(&buf[0], "\\", (LPVOID*)&pvi, (unsigned int*)&sz)) {
```

Comment:
```suggestion
            if (VerQueryValueA(buf.data(), "\\", &pvi, &sz)) {
```

If you make `sz` a `UINT` type, you don't need to cast